src/main/resources/static/script.js

```diff
@@ -1,11 +1,35 @@
-var map;
-
 function initMap() {
-  map = new google.maps.Map(document.getElementById("map"), {
-    center: { lat: -34.397, lng: 150.644 },
-    zoom: 8,
+  const directionsService = new google.maps.DirectionsService();
+  const directionsRenderer = new google.maps.DirectionsRenderer();
+  const map = new google.maps.Map(document.getElementById("map"), {
+    zoom: 7,
+    center: { lat: 41.85, lng: -87.65 },
```

Comment:
I'd suggest pulling these starting lat-long coordinates and zoom out and setting them as global variables at the start of the file. This way they aren't "magic numbers" in the code and can easily be found to edit in the future.